dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeMetadataConnector.java

```diff
@@ -270,6 +270,12 @@ public void addTasksTo(
           WarehousesFormat.Header.class,
           "SHOW WAREHOUSES",
           new TaskVariant(WarehousesFormat.AU_ZIP_ENTRY_NAME, AU));
+
+      addSingleSqlTask(
+          out,
+          FunctionsViewFormat.Header.class,
+          "SHOW FUNCTIONS",
```

Comment:
why not SHOW USER FUNCTIONS?

Reply:
We would want access to functions from all users, regardless of who is running the query.